tests/rms/test_models.py

```diff
@@ -41,3 +40,23 @@ def test_dict_model_functionality():
 
     obj["hpc_resources"]["num_cores_per_node"] = 2
     assert obj["hpc_resources"].get("num_cores_per_node", None) == 2
+
+
+def test_object_functionality():
```

Comment:
This file is under the rms subfolder, while you're adding a test for jms. Please move it to tests/jms/test_resources.py

Reply:
Move test to jms folder